### ISSUE TITLE
Add to CRS Registry official reference system of the Republic of Croatia EPSG:3765

### ISSUE THREAD

**Issue by Ana-SGA** [INSPIRE-MIF/helpdesk#83]
Dear all,

we have already commented the addition of the official coordinate reference system of the Republic of Croatia to the CRS Registry - EPSG:3765. 

Since other countries are also opening issues here with requests to add their reference systems, please also include Croatian reference system EPSG:3765 to the CRS Registry (otherwise validation fails).

Thanks!

Kind regards, 
Ana

**Comment by sMorrone** [comment]
Dear @Ana-SGA 

> we have already commented the addition of the official coordinate reference system of the Republic of Croatia to the CRS Registry - EPSG:3765

I am adding here reference to abovementioned discussion [#38](https://github.com/INSPIRE-MIF/helpdesk/issues/38)
So, if I get it well, this issue is a placeholder to remember to propose the addition of EPSG:3765 to the control body of the CRS registry as soon as that registry is in place. Is this correct?

**Comment by Ana-SGA** [comment]
Dear @sMorrone ,

yes, that is correct - we commented somewhere that I should open an issue for EPSG:3765 so that all requests for adding other CRSs would be in one place.

And just to check - is anything other needed on our part to initiate the procedure and include our Official coordinate reference sysytem?

**Comment by sMorrone** [comment]
>  is anything other needed on our part to initiate the procedure and include our Official coordinate reference sysytem?

no, at least not for the moment being. To my knowledge, the details on how proposal of additional CRS will need to be submitted to the MIG (e.g., if request posted in this helpdesk could be already sufficient) still needs to be provided /shared. But for sure we will come back with more details as soon as we have.
As additional info, I would like to point you [to this comment ](https://github.com/INSPIRE-MIF/helpdesk/issues/5#issuecomment-765500938) related to requirement for the endorsement of additional CRS.

**Comment by Ana-SGA** [comment]
Dear @sMorrone ,

thanks for the information and please if you will have any new details keep us posted.

Kind regards,
Ana
